src/lsp/cobol_cfg/cfg_builder.mli

```diff
@@ -0,0 +1,21 @@
+(******************************************************************************)
+(*                                                                            *)
+(*     Copyright (c) 2021-2023 OCamlPro SAS                                   *)
+(*                                                                            *)
+(*     All rights reserved.                                                   *)
+(*     This file is distributed under the terms of the                        *)
+(*     OCAMLPRO-NON-COMMERCIAL license.                                       *)
+(*                                                                            *)
+(******************************************************************************)
+
+type graph = {
+  name: string;
+  string_repr_dot: string;
+  string_repr_d3: string;
+  nodes_pos: (int * Cobol_ptree.srcloc) list
+}
```

Comment:
For later: this `graph` should actually be the representation as in `ocamlgraph`. The export to graphviz & d3 should be part of the `cobol_lsp` library itself. The `srcloc`s can probably be attached as node label/attributes in the `ocamlgraph` representation, and then extracted into this list in the code that build the reply in `cobol_lsp`.